portfolio/src/main/webapp/comment-page/comments.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>Comments</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="../style.css">
```

Comment:
Is there a particular reason why  ../style    vs   style?

Reply:
It's because I moved some files around into new packages. Doing ../style gets the file in the above directory (because the style.css file is outside of the current folder).